modules/openapi-generator/src/main/resources/typescript-nestjs/api.service.mustache

```diff
@@ -32,9 +31,9 @@ export class {{classname}} {
     public defaultHeaders = new Map()
     public configuration = new Configuration();
 
-    constructor(protected httpClient: HttpService, configuration: Configuration) {
+    constructor(protected httpClient: HttpService, @Optional() configuration: Configuration) {
         this.configuration = configuration;
```

Comment:
I think it would be better to set the configuration only if it is provided, to avoid null checks in the rest of the code
```suggestion
        this.configuration = configuration || this.configuration;
```